tests/httpwrappers/tests.py

```diff
@@ -299,6 +300,18 @@ def test_fromkeys_noniterable(self):
             QueryDict.fromkeys(0)
 
 
+class HttpResponseBaseTests(SimpleTestCase):
+    def test_charset_setter(self):
+        r = HttpResponseBase()
+        r.charset = "utf-8"
+        self.assertEqual(r.charset, HttpResponseBase(charset="utf-8").charset)
+
+    def test_reason_phrase_setter(self):
+        r = HttpResponseBase()
+        r.reason_phrase = "test"
+        self.assertEqual(r.reason_phrase, HttpResponseBase(reason="test").reason_phrase)
```

Comment:
Do we need new objects? It think it's enough to check a value:
```suggestion
        self.assertEqual(r.reason_phrase, "test")
```